### PR TITLE
Run weekly tests against 20.09 and add slash commands

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by mvdbeek** [comment]
If merged we can trigger a test run with
```
/run-all-tests
```

as the command. Triggering this is restricted to users that have write permissions on the repo (but this can be configured if necessary).

**Comment by mvdbeek** [comment]
I've run an example on my fork [here](https://github.com/mvdbeek/tools-iuc/issues/9)

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
     # Run at midnight every monday
     - cron: '0 0 * * 1'
   repository_dispatch:
-    types: [run-all-tool-tests]
+    types: [run-all-tool-tests-command]
```
Comment (bernt-matthias) [inline .github/workflows/ci.yaml:7]: Wondering why this has the suffix `-command`

Comment (bernt-matthias) [reply .github/workflows/ci.yaml:7]: Got it.

**Review by bernt-matthias — APPROVED** [review]

**Comment by bgruening** [comment]
Oh this is cool!

**Pull request merged**